dev_guide/quota.adoc

```diff
@@ -243,3 +243,9 @@ The `*resource-quota-sync-period*` setting is designed to balance system
 performance. Reducing the sync period can result in a heavy load on
 the master.
 ====
+
+[[accounting_quota_dc]]
+
+== Accounting for Quota in Deployment Configs
+
+If a quota has been defined for your project, read about the considerations for any Deployment Configs link:./deployments.html#deployment-resources[here].
```

Comment:
If a quota has been defined for your project, see link:../dev_guide/deployments.html#deployment-resources[Deployment Resources] for considerations on any deployment configurations.